### ISSUE TITLE
alias echo breaks git prompt

### ISSUE THREAD

**Issue by equwal** [woefe/git-prompt.zsh#31]
putting `alias echo="something else"` in your aliasrc breaks git-prompt. It shouldn't be doing alias expansion...

Test case:
Install plan9port
`alias echo="9 echo"`

It breaks by printing colour codes literally, with the test case.

**Comment by woefe** [comment]
The problem is that process substitution is subject to alias expansion and I don't see a way to turn it off. Following patch fixes aliasing `echo`, but if you `alias builtin=ls` you basically have the same problem again.

```diff
diff --git a/git-prompt.zsh b/git-prompt.zsh
index 2af01df..8a3c6dc 100644
--- a/git-prompt.zsh
+++ b/git-prompt.zsh
@@ -308,11 +308,11 @@ function _zsh_git_prompt_async_request() {
     # Fork a process to fetch the git status and open a pipe to read from it
     exec {_ZSH_GIT_PROMPT_ASYNC_FD}< <(
         # Tell parent process our pid
-        echo $sysparams[pid]
+        builtin echo $sysparams[pid]

         _zsh_git_prompt_git_status
         [[ -n "$ZSH_GIT_PROMPT_ENABLE_SECONDARY" ]] \
-            && echo -n "##secondary##" \
+            && builtin echo -n "##secondary##" \
             && _zsh_git_prompt_git_status_secondary
     )
```

**Comment by woefe** [comment]
Fixed as mentioned above in 22db871
